lib/capistrano/asg/aws/autoscaling.rb

```diff
@@ -9,6 +9,16 @@ module AutoScaling
         include Region
         include Capistrano::DSL
 
+        # class variable to only set 1 primary across multiple ASG's for an app
+        @@add_primary = true
```

Comment:
im not sure if cap runs multithreaded but if it does this wouldnt be threadsafe, not a big deal